ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -471,6 +471,7 @@ SS:
   - current_sensor
   - power_sensor
 
+  
```

Comment:
Please revert this if its just whitespace.

Reply:
This is just a white space...

Reply:
white space deleted